starknet_py/net/l1/contracts.py

```diff
@@ -64,8 +67,8 @@ def l2_to_l1_messages(
         )
 
     def l1_to_l2_messages(
-        self, msg_hash: int, block_number: EthBlockIdentifier = None
-    ) -> int:
+        self, msg_hash: bytes, block_number: EthBlockIdentifier = None
+    ) -> HexBytes:
```

Comment:
why not int here? we would just like to return the count of messages

Reply:
Updated this to bytes, count of messages is returned in messages.py, where this bytes response is cast to int.